pkg/resources/shared_database_acceptance_test.go

```diff
@@ -0,0 +1,175 @@
+	"github.com/stretchr/testify/require"
+)
+
+func TestAcc_CreateSharedDatabase_minimal(t *testing.T) {
+	name := acc.TestClient().Ids.Alpha()
+	comment := random.Comment()
+
+	newName := acc.TestClient().Ids.Alpha()
+	newComment := random.Comment()
+
+	configVariables := func(name string, shareName sdk.ExternalObjectIdentifier, comment string) config.Variables {
+		return config.Variables{
+			"name":       config.StringVariable(name),
+			"from_share": config.StringVariable(shareName.FullyQualifiedName()),
+			"comment":    config.StringVariable(comment),
+		}
+	}
+
+	shareExternalId := createShareableDatabase(t)
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.RequireAbove(tfversion.Version1_5_0),
+		},
+		PreCheck:     func() { acc.TestAccPreCheck(t) },
+		CheckDestroy: acc.CheckDestroy(t, resources.SharedDatabase),
+		Steps: []resource.TestStep{
+			{
+				ConfigVariables: configVariables(name, shareExternalId, comment),
+				ConfigDirectory: acc.ConfigurationDirectory("TestAcc_SharedDatabase/basic"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "name", name),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "from_share", shareExternalId.FullyQualifiedName()),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "external_volume", ""),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "catalog", ""),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "default_ddl_collation", ""),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "log_level", "OFF"),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "trace_level", "OFF"),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "comment", comment),
+				),
+			},
+			{
+				ConfigVariables: configVariables(newName, shareExternalId, newComment),
+				ConfigDirectory: acc.ConfigurationDirectory("TestAcc_SharedDatabase/basic"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "name", newName),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "from_share", shareExternalId.FullyQualifiedName()),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "external_volume", ""),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "catalog", ""),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "default_ddl_collation", ""),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "log_level", "OFF"),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "trace_level", "OFF"),
+					resource.TestCheckResourceAttr("snowflake_shared_database.test", "comment", newComment),
```

Comment:
missing import test

Reply:
I think it's sufficient to have one test (with all fields set) that checks the import step. At least in this case where the resource is pretty straight forward without any conditional logic. @sfc-gh-asawicki  wdyt?

Reply:
+1